### PR TITLE
hyperscan: Generate and install dynamic and static library files

### PR DESCRIPTION
**Pull request by ptoomey3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I needed the dynamic hyperscan libraries while doing some work with ctags, so here we are :smile:.

### CODE REVIEW

**Comment by ptoomey3** [comment]
Let me know if you would like the commit message amended with the current version of hyperscan..I wasn't clear if that was the commit pattern regardless of bumping versions in a formula or not.

File: Formula/hyperscan.rb [hunk 1]
```diff
@@ -22,6 +22,9 @@ class Hyperscan < Formula
   def install
     mkdir "build" do
       args = std_cmake_args
+      args += %w[
+        -DBUILD_STATIC_AND_SHARED=on
+      ]
```
Comment (s172262) [inline Formula/hyperscan.rb]: To write the following is faster in Ruby.
```
args << "-DBUILD_STATIC_AND_SHARED=on"
```

Comment (ptoomey3) [reply Formula/hyperscan.rb]: yup..was just trying to follow the pattern below: https://github.com/Homebrew/homebrew-core/pull/20054/files#diff-c6d2052c00252951884d401ab75abdceR35

Comment (s172262) [reply Formula/hyperscan.rb]: That is slower than the following. And following is more compact.
```
args << "-DCMAKE_BUILD_TYPE=Debug" << "-DDEBUG_OUTPUT=on"
```

Comment (ptoomey3) [reply Formula/hyperscan.rb]: Would you like me to update it? I was following the existing pattern since I didn't commit the original formula.

Comment (s172262) [reply Formula/hyperscan.rb]: I think original part is not your task.

Comment (DomT4) [reply Formula/hyperscan.rb]: @s172262 is correct with the first comment. When it's just a single argument don't use an array. If there are multiple arguments that cause the line to dip over 80 characters, use an array.

**Comment by ptoomey3** [comment]
Ok..updated in https://github.com/Homebrew/homebrew-core/pull/20054/commits/b601b61dbcd9431674ec21ec138db950272eb022.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @ptoomey3! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**